README.md

```diff
@@ -150,6 +150,30 @@ let props = doc.properties();
 let greeting = props.get("greeting");
 ```
 
+
+### Golang
+
+```go
+	// Open a database:
+	var config = &cblcgo.DatabaseConfiguration{Directory: "./", Flags: cblcgo.Database_Create}
+	db, err := cblcgo.Open("my_db", config)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	// Create a document:
+	doc := cblcgo.NewDocumentWithId("foo")
+	doc.Props["greeting"] = "Howdy!"
+	db.Save(doc, cblcgo.LastWriteWins)
+
+	// Read it back:
+	readDoc, _ := db.GetMutableDocument("foo")
+	readProps := readDoc.Props
+	greeting := readProps["greeting"]
+
+	fmt.Println("greeting:", greeting)
+```
+
```

Comment:
This README only gives examples for bindings that are included in this repo, so it's not appropriate to show Go here. Please remove this.